package.nls.cs.json

```diff
@@ -136,9 +136,9 @@
   "generateOptionsSchema.symbolOptions.moduleFilter.mode.description": "Určuje, v jakém ze dvou základních operačních režimů pracuje filtr modulu.",
   "generateOptionsSchema.symbolOptions.moduleFilter.mode.loadAllButExcluded.enumDescription": "Načte symboly pro všechny moduly, pokud není modul v poli „excludedModules“.",
   "generateOptionsSchema.symbolOptions.moduleFilter.mode.loadOnlyIncluded.enumDescription": "Nepokoušejte se načíst symboly pro ŽÁDNÝ modul, pokud není v poli „includedModules“, nebo je součástí nastavení „includeSymbolsNextToModules“.",
-  "generateOptionsSchema.symbolOptions.searchMicrosoftSymbolServer.description": "Pokud je hodnota true, přidá se do cesty pro hledání symbolů server symbolů pro produkty Microsoft (https​://msdl.microsoft.com​/download/symbols). Pokud tato možnost není zadaná, výchozí hodnota je false.",
-  "generateOptionsSchema.symbolOptions.searchNuGetOrgSymbolServer.description": "Pokud je hodnota true, přidá se do cesty pro hledání symbolů server symbolů pro NuGet.org (https​://symbols.nuget.org​/download/symbols). Pokud tato možnost není zadaná, výchozí hodnota je false.",
-  "generateOptionsSchema.symbolOptions.searchPaths.description": "Pole adres URL serveru symbolů (například: http​://MyExampleSymbolServer) nebo adresářů (například: /build/symbols) k vyhledávání souborů .pdb. Tyto adresáře budou prohledány kromě výchozích umístění – vedle modulu a cesty, kam byl soubor pdb původně přemístěn.",
+  "generateOptionsSchema.symbolOptions.searchMicrosoftSymbolServer.description": "Pokud je hodnota true, přidá se do cesty pro hledání symbolů server symbolů pro produkty Microsoft (https\u200b://msdl.microsoft.com\u200b/download/symbols). Pokud tato možnost není zadaná, výchozí hodnota je false.",
```

Comment:
Are these changes just going to get overwritten the next time a localization pass comes along?

Reply:
@Cosifne is there a way to indicate in the translation that a string should remain unchanged?  
That might not even be enough though, any tool in the localization pipeline could potentially get rid of zero-width spaces.  If these weren't package.json I'd suggest using the string.format version, but not sure how to fix this for package.json strings.

Reply:
@dibarbet in case you missed this, Andrew already has instructions telling the LOC team not to mess with them. But they are weird, so not surprisingly, sometimes the loc team doesn't follow the instructions.

Reply:
@gregg-miskelly It looks like the loc team kept the \u200b character but something in their pipeline might have escaped it. I think something in their pipeline needs to run `packageNlsJson.replace(/\u200b/gm, '\\u200b');`

Reply:
What I found earlier that sometimes the `\u200b` was converted to the Unicode character, which maybe isn't ideal because it isn't human readable, but it is okay. But also, sometimes they dropped it all together. If it is hard for the loc team to leave it escaped, I think it is fine to accept the former. But obviously we need something to prevent the later.

Reply:
I think there should a way to make a part of string unchanged.
See https://github.com/microsoft/vscode/issues/125774#issuecomment-1339954388
Can we try to lock the whole link?